autoscheduler/scraper/tests/api_tests.py

```diff
@@ -0,0 +1,253 @@
+from datetime import time
+from rest_framework.test import APITestCase, APIClient
+from scraper.models.course import Course
+from scraper.models.instructor import Instructor
+from scraper.models.section import Section, Meeting
+from scraper.serializers import CourseSerializer, SectionSerializer
+
+class APITests(APITestCase):
+    """ Tests API functionality """
+    def setUp(self):
+        self.client = APIClient()
+        self.courses = [
+            Course(id='123123', dept='CSCE', course_num='181',
+                   title='Introduction to Computing', term='201931', credit_hours=3),
+            Course(id='123124', dept='CSCE', course_num='315',
+                   title='Programming Studio', term='201931', credit_hours=3),
+            Course(id='123125', dept='COMM', course_num='203',
+                   title='Public Speaking', term='201831', credit_hours=3),
+            Course(id='123126', dept='COMM', course_num='203',
+                   title='Public Speaking', term='201931', credit_hours=3),
+            Course(id='123127', dept='LAW', course_num='7500S',
+                   title='Sports Law', term='202031', credit_hours=None),
+        ]
+        test_instructors = [
+            Instructor(id='Akash Tyagi'),
+            Instructor(id='John Moore'),
+        ]
+        for instructor in test_instructors:
+            instructor.save()
+        self.sections = [
+            Section(crn=12345, id='000001', subject='CSCE', course_num='310',
+                    section_num='501', term_code='201931', min_credits='3',
+                    honors_only=False, web_only=False, max_enrollment=50,
+                    current_enrollment=40, instructor=test_instructors[0]),
+            Section(crn=12346, id='000002', subject='CSCE', course_num='310',
+                    section_num='502', term_code='201931', min_credits='3',
+                    honors_only=False, web_only=False, max_enrollment=50,
+                    current_enrollment=40, instructor=test_instructors[1],
+                    instructor_gpa=3.2),
```

Comment:
Shouldn't these sections correspond to a Course that's in the DB? I'd change the `course_num` from 310 to 315. Not sure if this will change anything in the tests, but logically that's what it should be.